File: server/src/main/java/com/remarkablesystems/childwatch/domain/food/MealType.java
package com.remarkablesystems.childwatch.domain.food;

import java.util.Arrays;
import java.util.List;

public enum MealType {
	BREAKFAST,
	AM_SNACK,
	LUNCH,
	PM_SNACK,
	SUPPER,
	EV_SNACK;
	
	public static List<MealType> ALL = Arrays.asList(MealType.BREAKFAST, MealType.AM_SNACK,MealType.LUNCH,MealType.PM_SNACK,MealType.SUPPER,MealType.EV_SNACK );

}

File: client/src/app/food/model/meal-type.ts
export class MealType {
  public ALL = ['BREAKFAST', 'AM_SNACK', 'LUNCH', 'PM_SNACK', 'SUPPER', 'EV_SNACK'];
  public toDisplay(type: string): string {
    switch (type) {
      case 'BREAKFAST': return 'Breakfast';
      case 'AM_SNACK': return 'AM Snack';
      case 'LUNCH': return 'Lunch';
      case 'PM_SNACK': return 'PM Snack';
      case 'SUPPER': return 'Supper';
      case 'EV_SNACK': return 'Evening Snack';
      default: return undefined;
    }
  }

}